vertx-web/src/main/java/io/vertx/ext/web/handler/sockjs/impl/WebSocketTransport.java

```diff
@@ -122,7 +123,7 @@
     public void sendFrame(final String body) {
       if (log.isTraceEnabled()) log.trace("WS, sending frame");
       if (!closed) {
-        ws.writeFrame(WebSocketFrame.textFrame(body, true));
+        ws.writeBinaryMessage(Buffer.buffer(body));
```

Comment:
Switching from text frames to binary frames is not compatible with sockjs clients as I discovered.